website/docs/index.html.markdown

```diff
@@ -179,13 +179,19 @@ In addition to [generic `provider` arguments](https://www.terraform.io/docs/conf
 
 For more information on configuring and managing programmatic API Keys see the [MongoDB Atlas Documentation](https://docs.atlas.mongodb.com/tutorial/manage-programmatic-access/index.html).
 
-## HashiCorp Terraform Version Requirement
-| MongoDB Atlas Provider version  | Required Terraform version |
-| ------------- | ------------- |
-| `1.12.0` and above  | `1.0+`  |
-| below `1.12.0`  | `0.13+` |
-
-* While we support minimum of HashiCorp Terraform version 1.0+ (as we have migrated to [Plugin Protocol Version 6](https://developer.hashicorp.com/terraform/plugin/terraform-plugin-protocol#protocol-version-6)), for your safety we strongly recommend only consuming versions of Terraform that are currently receiving Security/Maintenance Updates. For latest Terraform versions see: https://endoflife.date/terraform 
+## [HashiCorp Terraform Version](https://www.terraform.io/downloads.html) Compatibility Matrix
```

Comment:
my feeling is that this info is very below in the page but i suppose it doesn't fit more above

Reply:
played around with the spacing, hope this helps with readability